### PR TITLE
schema for ite2 

### PR DESCRIPTION
**Pull request by maknontawat**
check if the schema are good and i didn't missed any important valuable

### CODE REVIEW

File: db/schema1.sql [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
 CREATE TABLE travelActi (
-planid int REFERENCES travelPlan,
-actiname varchar(50),
-actidesp text
+	planid int REFERENCES travelPlan(planid),
+	actiName varchar(50) NOT NULL,
+	actiStart date DEFAULT 0000-00-00,
+	actiEnd date DEFAULT 0000-00-00,
+	actiLocation varchar(100),
+	actiPhoto varchar(100), /*store the URl for the photo (only 1 photo for each acti)*/
```
Comment (swa181) [inline db/schema1.sql]: Might have to revisit this.
Urls can be very long.

Comment (maknontawat) [reply db/schema1.sql]: make it to text?

Comment (swa181) [reply db/schema1.sql]: if it works the same but without the limitations, then great

File: db/schema1.sql [hunk 2]
```diff
@@ -1,28 +1,34 @@
...
+	fname varchar(50) NOT NULL,
+	lname varchar(50) NOT NULL
 );
 
-
 CREATE TABLE travelPlan (
-planid serial PRIMARY KEY,
-plantitle varchar(50) NOT NULL,
-leader varchar(50)
+	planid serial PRIMARY KEY,
```
Comment (swa181) [inline db/schema1.sql]: i think you can just name these to id and title

File: db/schema1.sql [hunk 3]
```diff
@@ -1,28 +1,38 @@
...
+	plantitle varchar(50) NOT NULL
 );
 
 CREATE TABLE travelActi (
```
Comment (swa181) [inline db/schema1.sql]: can you change it to travelactivity

File: react-ui/src/components/plans-dashboard/Dashboard.js [hunk 4]
```diff
@@ -61,17 +61,29 @@ class Dashboard extends React.Component {
...
 					<div id="CalendarWrapper">
 						<div style={paginationStyle} onClick={this.renderPlanPicker}>
-							<h1 style={paginationStyle}>My Plans </h1>
-						</div>
-						<h1 style={paginationStyle}>> {this.state.currentCalendar}</h1>
+							<button style={paginationStyle}> My Plans </button>
+						</div><br/>
+						<h1 style={paginationStyle}> Plan -> {this.state.currentCalendar}</h1><br/>
+						<button> Add authors </button>
+						<h3> \O/ </h3>
```
Comment (swa181) [inline react-ui/src/components/plans-dashboard/Dashboard.js]: whats this for

**Pull request merged**